### ISSUE TITLE
Task operations for initial Apache airflow DAG

### ISSUE THREAD

**Issue by krutarth4** [amosproj/amos2023ws04-pipeline-manager#42]
## User story
1. Backend could initiate a task 
2. I want some result or output from a pipeline like word-count
3. So that logged-in user can see the processed information. 

## Acceptance criteria
* Pipeline can execute the tasks successfully without any error 

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the main branch
* All acceptance criteria were met
* Product owner approved features
* Developers agreed to release

**Comment by krutarth4** [comment]
@Elementator and @eubec  could you take a look at the issue and assign it to me. Make changes if you think is important.

**Comment by krutarth4** [comment]
@eubec  and @Elementator  Hey this issue was finished last sprint. 

The acceptance crietria was wrong , It was that the pipeline is able to get access to a csv file and analze it + could give a wordcount result

**Comment by Elementator** [comment]
new issue, receice result in backend
